### DISCUSSION TITLE
Documentation - for which version?

### DISCUSSION THREAD

**Discussion by chudoba** [DIRACGrid/DIRAC#6857] [Q&A] [answered]
Hi, 

I tried the  dirac-dms-pfn-metadata command. According the documentation here:
https://dirac.readthedocs.io/en/latest/UserGuide/HowTo/DataManagement/index.html#how-to-have-the-metadata-of-an-file
it should work with or without SE specification. However I get:
 [chudoba@ui1 ~]$ dirac-dms-pfn-metadata  /auger/prod/d0002/helium/16.5_17.0/log/test/t.5
Error when parsing command line arguments: not all mandatory arguments are defined.

Retrieve metadata for a PFN given a valid DIRAC SE

Usage:
  dirac-dms-pfn-metadata [options] ... PFN SE
...

I suspect that the documentation is not valid for my DIRAC version. I use the one from CVMFS:
/cvmfs/dirac.egi.eu/dirac/bashrc_egi
$ dirac-version
8.0.3

How can I check which version is described in the documentation?

Thanks,
Jiri.

PS: the command above works OK if I specify an SE:
$ dirac-dms-pfn-metadata  /auger/prod/d0002/helium/16.5_17.0/log/test/t.5 CESNET-disk
{'Failed': {},
 'Successful': {'/auger/prod/d0002/helium/16.5_17.0/log/test/t.5': {'Accessible': True,
                                                                    'Checksum': '',
                                                                    'Directory': False,
...

and even if the SE is not up:
$ dirac-dms-pfn-metadata  /auger/prod/d0002/helium/16.5_17.0/log/test/t.5 UNAM-disk
{'Failed': {'/auger/prod/d0002/helium/16.5_17.0/log/test/t.5': 'Communication '
                                                               'error on send '
                                                               '( 70 : Failed '
                                                               'to retrieve '
                                                               'metadata '
                                                               "GError('srm-ifce "
....

**Comment by fstagni** [comment, marked as answer]
The command works for me using DIRAC v8.0.15. 

The documentation is built for different versions:

![image](https://user-images.githubusercontent.com/785576/221587233-f3236de0-932c-4673-bc61-9fdc1cd3b3be.png)



and the code documentation is built on the latest tag:

![image](https://user-images.githubusercontent.com/785576/221586749-ae43e95c-bc76-4c28-b94d-c75d7b524dcb.png)
